### PR TITLE
[RHACS] ROX-20486: Adding updates to release notes for 4.2.3 patch release

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.2

Issue:
https://issues.redhat.com/browse/ROX-20486

Link to docs preview:
https://67617--docspreview.netlify.app/openshift-acs/latest/release_notes/42-release-notes#resolved-in-version-4203_release-notes-42

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-4.2` 
- Reference PR: https://github.com/openshift/openshift-docs/pull/67393

### CODE REVIEW

**Comment by agantony** [comment]
Tech review needed

File: release_notes/42-release-notes.adoc [hunk 1]
```diff
@@ -440,6 +441,17 @@ It contains the following bug fixes and changes:
...
 
+
+[id="resolved-in-version-4203_{context}"]
+=== Resolved in version 4.2.3
+
+*Release date:* 15 November 2023
+
+This release of {product-title-short} includes updates to {op-system-base-full} base images and includes the following fixes:
+
+* All containers have been rebuilt and now include container CVE fixes for link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487]: Flaw in handling multiplexed streams in the HTTP/2 protocol and link:https://access.redhat.com/security/cve/CVE-2023-40217[CVE-2023-40217]: Python 3 ssl.SSLSocket vulnerability.
```
Comment (vladbologa) [inline release_notes/42-release-notes.adoc:452]: The 4.2.2 containers are currently not marked as being affected by CVE-2023-44487.

**Review by vladbologa — CHANGES_REQUESTED** [review]

**Review by vladbologa — APPROVED** [review]
LGTM

**Review by mjpytlak — COMMENTED** [review]
@agantony Please squash your commits. Also - I presume you do not want this merged until November 15th?

**Comment by agantony** [comment]
> @agantony Please squash your commits. Also - I presume you do not want this merged until November 15th?

Oh yeah, that's for November 15, good catch!
I will squash the commits too.

**Comment by mjpytlak** [comment]
@agantony I will be off the squad next week, but for the sake expediency remind me on the 15th to merge this. Have a good weekend.

File: release_notes/42-release-notes.adoc [hunk 2]
```diff
@@ -440,6 +441,17 @@ It contains the following bug fixes and changes:
...
+
+[id="resolved-in-version-4203_{context}"]
+=== Resolved in version 4.2.3
+
+*Release date:* 15 November 2023
+
+This release of {product-title-short} includes updates to {op-system-base-full} base images and includes the following fixes:
+
+* link:https://access.redhat.com/security/cve/CVE-2023-40217[CVE-2023-40217]: Python 3 ssl.SSLSocket vulnerability.
+* The HTTP/2 functionality in the {product-title-short} Operator webhook has been disabled to mitigate CVE-2023-44487.
```
[the remark this answers is not in the document]
Comment (agantony) [reply release_notes/42-release-notes.adoc:453]: I appreciate and value your feedback. However, let us leave it the way it is to maintain consistency with  https://docs.openshift.com/acs/4.1/release_notes/41-release-notes.html#resolved-in-version-415_release-notes-41:~:text=The%20HTTP/2%20functionality%20in%20the%20RHACS%20Operator%20webhook%20has%20been%20disabled%20to%20mitigate%20CVE%2D2023%2D44487.

**Review by kcarmichael08 — APPROVED** [review]
LGTM!

**Comment by agantony** [comment]
The GA date is yet to be confirmed. Please do not merge this PR.

File: release_notes/42-release-notes.adoc [hunk 3]
```diff
@@ -440,6 +441,18 @@ It contains the following bug fixes and changes:
...
 
+
+[id="resolved-in-version-4203_{context}"]
+=== Resolved in version 4.2.3
+
+*Release date:* 15 November 2023
+
+This release of {product-title-short} includes updates to {op-system-base-full} base images and includes the following fixes:
+
+* link:https://access.redhat.com/security/cve/CVE-2023-40217[CVE-2023-40217]: Python 3 ssl.SSLSocket vulnerability.
```
Comment (vladbologa) [inline release_notes/42-release-notes.adoc:452]:
```suggestion
```

Please remove this, 4.2.2 is not affected by CVE-2023-40217

**Review by vladbologa — CHANGES_REQUESTED** [review]

**Review by vladbologa — APPROVED** [review]

**Comment by kcarmichael08** [comment]
replaced by #68436

**Pull request closed without merging**